R/req-headers.R

```diff
@@ -9,6 +9,10 @@
 #'   * Use `NULL` to reset a value to httr's default
 #'   * Use `""` to remove a header
 #'   * Use a character vector to repeat a header.
+#' @param .redact If `TRUE`, all the added headers are redacted. Can also be a
```

Comment:
What about making this a character vector? It makes it slightly harder to use in the `.redact = TRUE` case, but is easier if you only want to redact a single header, and I think a storing character vector will make for a much simpler implementation.

Reply:
I don't think supporting `TRUE` is needed IMO. I think it will most be used by package authors, and it's nice to be explicit.